Automattic/vip-go-mu-plugins
# Update Jetpack-9.3 subtree to the final 9.3 release

## Description

Update `jetpack-9.3` subtree to the final Jetpack 9.3 release.

## Changelog Description

### Plugin Updated: Jetpack 9.3

We have upgraded Jetpack 9.3 to the final Jetpack 9.3 release.

Although this new version will be rolled out to all sites on Thursday 14th, remember you can start testing it in your site by adding this constant to your `vip-config.php` file: `define( 'VIP_JETPACK_PINNED_VERSION', '9.3');`

## Checklist


## Steps to Test

1. Check out PR.
1. Add constant `define( 'VIP_JETPACK_PINNED_VERSION', '9.3' );`
1. Go to the Jetpack dashboard in wp-admin and ensure that the version is `9.3`
1. Ensure everything works as expected
